src/PrestaShopBundle/Entity/Repository/StockManagementRepository.php

```diff
@@ -75,21 +76,6 @@ abstract class StockManagementRepository
      */
     protected $tablePrefix;
 
-    /**
-     * @var int
-     */
-    protected $languageId;
-
-    /**
-     * @var int
-     */
-    protected $shopId;
-
-    /**
-     * @var \Context
-     */
-    protected $context;
-
```

Comment:
Could you document the BC break for the removal of these attributes  ?

Reply:
Same...BC would break only if the class is extended outside of core components.

Reply:
It's possible to extend legacy